bin/sandbox

```diff
@@ -24,12 +24,6 @@ case "${DB}" in
     ;;
 esac
 
-if [[ -n "${SOLIDUS_BRANCH}" ]]; then
```

Comment:
Could we add more information to the message of this commit? I think we're doing several things here:

- Installing after solidus
- Cleaning unneeded gems
- Acknowledging that solidus_starter_frontend's main branch only works for solidus master branch

Reply:
Ideally, we should split it in different commits, but a better explanation can be enough 🙂

Reply:
Thanks!